package.json

```diff
@@ -50,6 +51,76 @@
     "chai": "^3.5.0",
     "vscode-textmate": "^2.1.1"
   },
+  "runtimeDependencies": [
```

Comment:
The package manager does not (yet) allow packages to be specified for particular runtime IDs. This would need to be added in order to enable this scenario. I'll look at adding this.